src/taxes/urls.ts

```diff
@@ -1,17 +1,25 @@
+import { Dialog } from "@saleor/types";
+import { stringifyQs } from "@saleor/utils/urls";
 import urlJoin from "url-join";
 
+import { encodeURIComponentOptional } from "./utils/utils";
+
+export type TaxTab = "channels" | "countries" | "tax-classes";
+
 export const taxSection = "/taxes/";
-export const taxTabSectionUrl = (tab: string) => urlJoin(taxSection, tab);
+export const taxTabPath = (tab: TaxTab) => urlJoin(taxSection, tab);
+
+export type TaxesUrlDialog = "add-country";
+export type TaxesUrlQueryParams = Dialog<TaxesUrlDialog>;
+
+export const channelsListPath = (id?: string) =>
```

Comment:
Something like this? I'd like to maintain global uniqueness of path names

```suggestion
export const taxesInChannelsListPath = (id?: string) =>
```